browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1116,21 +1116,9 @@ Are you sure you want to do this?
       <message name="IDS_USER_WALLET_STATUS_NOT_CONNECTED">
         Not connected
       </message>
-      <message name="IDS_USER_WALLET_STATUS_CONNECTED">
-        Connected
-      </message>
       <message name="IDS_USER_WALLET_STATUS_VERIFIED">
         Verified
       </message>
-      <message name="IDS_USER_WALLET_STATUS_DISCONNECTED_NOT_VERIFIED">
```

Comment:
I think you'll still need this (`IDS_USER_WALLET_STATUS_DISCONNECTED_NOT_VERIFIED`) text, no?

Reply:
@szilardszaloki we don't need this text. It was only being used for WalletStatus string text for debugging.